src/adapters/TelnetAdapter.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('TelnetAdapter ', () => {
 
     describe('getExpressionDetails', () => {
         it('correctly handles both types of line endings', () => {
-            expect(adapter.getExpressionDetails(
+            expect(adapter['getExpressionDetails'](
```

Comment:
These all changed to string indexing because i made the methods private, so this satisfies the typescript errors.